File: Mobile/MyApplication/app/src/main/java/com/example/ooar/discountproject/model/NotificationProduct.java
package com.example.ooar.discountproject.model;

/**
 * Created by Onur Kuru on 5.3.2016.
 */
public class NotificationProduct {

    private Integer id;
    private Notification notificationId;
    private Product productId;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Notification getNotificationId() {
        return notificationId;
    }

    public void setNotificationId(Notification notificationId) {
        this.notificationId = notificationId;
    }

    public Product getProductId() {
        return productId;
    }

    public void setProductId(Product productId) {
        this.productId = productId;
    }
}

File: Web/src/model/NotificationProduct.js
var Notification = require("../model/Notification");
var Product = require("../model/Product");

var NotificationProduct = project.db.define('NotificationProduct', {
  id: {type: 'serial'}
});

NotificationProduct.hasOne('Notification', Notification, {field: 'notificationId'});
NotificationProduct.hasOne('Product', Product, {field: 'productId', autoFetch: true});

module.exports = NotificationProduct;